src/rpc/budget.cpp

```diff
@@ -436,6 +436,8 @@ UniValue getbudgetprojection(const JSONRPCRequest& request)
         throw std::runtime_error(
             "getbudgetprojection\n"
             "\nShow the projection of which proposals will be paid the next cycle\n"
+            "\nProposal fee tx time need to be +24hrs old from the current time. (Testnet is 5 mins)\n"
+            "\nNet Votes needs to be above Masternode Count divided by 10\n"
```

Comment:
initial `\n` in this two lines will add a extra jump line, you are already jumping a line at the end of the sentence. Both lines should be next to the previous one.

Reply:
Fixed